src/main/java/racingcar/util/CarNameValidator.java

```diff
@@ -0,0 +1,26 @@
+package racingcar.util;
+
+import racingcar.exception.CarNameException;
+
+public class CarNameValidator {
+
+  private static final int MAX_CAR_NAME_LENGTH = 5;
+
+  public static void validateName(String name) {
+    isProper(name);
+    isEmpty(name);
+  }
+
+  private static void isProper(String name) {
+    if (name.length() > MAX_CAR_NAME_LENGTH) {
+      throw new CarNameException("자동차 이름은 5자를 초과할 수 없습니다.");
```

Comment:
상수로 선언하신 부분을 이렇게 활용해보는건 어떠실까요~ ?🤔
```suggestion
      throw new CarNameException("자동차 이름은 "+ MAX_CAR_NAME_LENGTH +"자를 초과할 수 없습니다.");
```